conf/schemas/cloudtrail.json

```diff
@@ -22,26 +22,31 @@
   "cloudtrail:events": {
     "schema": {
       "additionalEventData": {},
+      "addendum": {},
       "apiVersion": "string",
       "awsRegion": "string",
       "errorCode": "string",
       "errorMessage": "string",
+      "eventCategory": "string",
       "eventID": "string",
       "eventName": "string",
       "eventSource": "string",
       "eventTime": "string",
       "eventType": "string",
       "eventVersion": "string",
+      "insightDetails": {},
```

Comment:
```
insightDetails
Insights events only. Shows information about the underlying triggers of an Insights event, such as event source, user agent, statistics, API name, and whether the event is the start or end of the Insights event. For more information about the contents of the insightDetails block, see CloudTrail Insights insightDetails element.

Since: 1.07

Optional: False
```